### ISSUE TITLE
Trouble compiling using Spago

### ISSUE THREAD

**Issue by arunvickram** [purescript-concur/purescript-concur-react#33]
This is similar to #29, but I'm also having trouble compiling using Spago: 

```
purs compile: No files found using pattern: .spago/concur-react/v0.3.9/src/**/*.purs
```

I'm unsure if this is specifically due to the spago config in the repo. I tried overriding this in my `packages.dhall`:

```dhall
let overrides =
      { concur-react =
          upstream.concur-react // { sources = ["lib/src/**/*.purs"] }
      , react =
          upstream.react // { version = "v8.0.0" }
      }
```

Been pulling my hair out for a day trying to figure this out, would appreciate some insight on this. Thanks!

**Comment by ajnsit** [comment]
Sorry, this is exactly the same issue as #29. I added a new backend to Concur, and split out the core library, and as https://github.com/spacchetti/spago/issues/288 shows, spago does not support publishing multiple libraries from the same repository well.

I'll push a fix in a minute.

**Comment by ajnsit** [comment]
@arundilipan <s>This should be fixed in https://github.com/ajnsit/purescript-concur/commit/ac23d7d151d0105595c18df8e093e219513f43e3, can you please let me know if it still doesn't work?</s>

**Comment by ajnsit** [comment]
@arundilipan Ugh, I just realised that it still won't work. I will need to move the core library, and the various backends into their own repos. Will make that change soon.

**Comment by arunvickram** [comment]
@ajnsit Thanks for the update, I figured that may have been the case because I couldn't override the `sources` flag for `concur`. What do you suggest I do in the meantime? Potentially use `react-basic-hooks` for now and use `concur` at a later time?

**Comment by ajnsit** [comment]
Oh you can simply use an earlier version of the library from before I made the change. An example of a working configuration is at https://github.com/FPNCR/fpncr.github.io/blob/master/packages.dhall

The override configuration there is as below. The dependencies section is not needed I think.

```
let overrides =
  { concur-react =
    upstream.concur-react
    // { dependencies =
         [ "aff"
         , "arrays"
         , "avar"
         , "console"
         , "foldable-traversable"
         , "free"
         , "nonempty"
         , "profunctor-lenses"
         , "react"
         , "react-dom"
         , "tailrec"
         , "web-dom"
         , "web-html"
         ]
       , repo =
         "https://github.com/ajnsit/purescript-concur.git"
       , version =
         "fe68e5cdb7e214e087dc070d9650988e8d64cd42"
       }
  , react =
      upstream.react // { version = "v8.0.0" }
  }
```

**Comment by arunvickram** [comment]
Okay thanks! Could you potentially leave that in the documentation for now until you make the move?

**Comment by ajnsit** [comment]
Sorry for the delay. I have published versions 0.4.1 for both purescript-concur-react, and purescript-concur-core to pursuit. They should be in the package-sets soon.

Please feel free to reopen this if it still doesn't work for you.
